src/Symfony/Component/Config/Definition/ExpressionNode.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Definition;
+
+use Symfony\Component\Config\Definition\Exception\InvalidTypeException;
+use Symfony\Component\ExpressionLanguage\Expression;
+
+/**
+ * This node represents a Symfony Expression Language Expression.
+ *
+ * @author Magnus Nordlander <magnus@fervo.se>
+ */
+class ExpressionNode extends VariableNode
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function validateType($value)
+    {
+        if (!$value instanceof Expression) {
+            $ex = new InvalidTypeException(sprintf(
```

Comment:
On one line, $ex could be $e, to be consistent with other places in the code base.